sktime/forecasting/ets.py

```diff
@@ -238,7 +239,12 @@ def __init__(
         self.n_jobs = n_jobs
 
         super(AutoETS, self).__init__(random_state=random_state)
-
+        
+        if self.auto and (any(param for param in [trend, damped_trend, seasonal]) or error=="add"):
+            warnings.warn(
+                "Warning: The user-specified parameters provided alongside auto=True in AutoETS may not be respected. The AutoETS function automatically selects the best model based on the information criterion, ignoring the error, trend, seasonal, and damped parameters when auto=True is set. Please ensure that your intended behavior aligns with the automatic model selection."
```

Comment:
lint comment: this line is quite long, and my guess is that it'll trigger code quality check, especially `flake8` etc. Can we split this into multiple lines?

E.g. these two are equivalent:

```python3
text_1 = "assume it is a long sentence"
text_2 = (
    "assume it is"
    " a long "
    "sentence"
)

assert text_1 == text_2
```